tests/test_build.nf.test

```diff
@@ -0,0 +1,36 @@
+nextflow_pipeline {
+
+    name "Test pipeline"
+    script "../main.nf"
+    profile "test_build"
+    tag "pipeline"
+    tag "pipeline_rnafusion"
+    tag "test_build"
+
+    test("test_build") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+                no_cosmic  = true
+            }
+        }
```

Comment:
I would pass the params in the profile, to be more clear, because you now have 2 sources of params:
- the test_build profile.config itself
- the nf-test
I would expect the nf-test to run exactly as the test profile, so I would add all necessary parameters in the config.
This comment applies to all nf-tests

Reply:
Makes sense!